### PR TITLE
Arnej/test hex tensor feed

### PR DESCRIPTION
**Pull request by arnej27959**
I confirm that this contribution is made under the terms of the license found in the root directory of this repository's source tree and that I have the authority necessary to make this contribution on behalf of its copyright owner.

@geirst please review
@jobergum @lesters FYI

### CODE REVIEW

**Comment by jobergum** [comment]
Does it make sense for float as well?

File: tests/search/tensor_feed/tensor_feed_hex.rb [hunk 1]
```diff
@@ -0,0 +1,173 @@
+
```
Comment (geirst) [inline tests/search/tensor_feed/tensor_feed_hex.rb:1]: Please add copyright.

File: tests/search/tensor_feed/tensor_feed_hex.rb [hunk 2]
```diff
@@ -0,0 +1,173 @@
...
+  include TensorFeedTestBase
+
+  def setup
+    set_owner("arnej")
+    @base_dir = selfdir + "tensor_hex/"
+  end
+
+  def test_tensor_hex_feed
+    set_description("Test feeding of tensor field and retrieval via search and visit")
```
Comment (geirst) [inline tests/search/tensor_feed/tensor_feed_hex.rb:15]:
```suggestion
    set_description("Test hex format feeding of tensor field and retrieval via search and visit")
```

**Review by geirst — APPROVED** [review]
👍

**Comment by arnej27959** [comment]
> Does it make sense for float as well?

maybe, maybe not.  Each float would need 8 hex digits, which may be more or less text than normal json floats, depending on how much precision etc they have.  I'm not sure what the difference in cost of parsing would be.

**Pull request merged**